### ISSUE TITLE
Alpine 3.4

### ISSUE THREAD

**Issue by ashleysommer** [mhart/alpine-node#45]
Just a quick question.
Why is the alpine 3.4 base image only used for the node 6.2.1 build, and not for the 5.x, 4.x, 0.12, and 0.10 versions?
I can see it is deliberately moved back to 3.3 for those versions, so I'm guessing there is a good reason.

**Comment by polarathene** [comment]
@ashleysommer this is mentioned in the [README](https://github.com/mhart/alpine-node#caveats) where the addition was added 14 days ago:

```
Tags 6.x of this image now use Alpine 3.4 – earlier versions use Alpine 3.3 although this may change if there are no issues reported with 3.4
```

Given time the other images will likely get updated.

**Comment by mhart** [comment]
Yeah, that's right – I don't want to change any existing vx.x.x tags (v6.2.1 was released after Alpine 3.4, all the rest were before) – but I'm going to update all versions on the next Node.js release, which should be next week.
